backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc

```diff
@@ -12,3 +12,7 @@ include::modules/dr-restoring-cluster-state-about.adoc[leveloffset=+1]
 
 // Restoring to a previous cluster state
 include::modules/dr-restoring-cluster-state.adoc[leveloffset=+1]
+
+.Additional resources
```

Comment:
Please add a unique anchor ID for `Additional resources` (see [Documentation guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies))